.appveyor.yml

```diff
@@ -14,6 +14,12 @@ environment:
   # Create expected SHELL variable for pipenv.
   SHELL: "windows"
   matrix:
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
```

Comment:
I have shuffled the order of the build matrix such that the `ANACONDA_TESTS_ONLY` are run first on Appveyor. We might want to mirror the ordering on Travis later on though.